ext/standard/html.c

```diff
@@ -1559,3 +1560,125 @@ PHP_FUNCTION(get_html_translation_table)
 	}
 }
 /* }}} */
+
+static zend_result cache_request_parse_body_option(HashTable *options, zval *option, int cache_offset)
+{
+	if (option) {
+		zend_long result;
+		if (Z_TYPE_P(option) == IS_STRING) {
+			zend_string *errstr;
+			result = zend_ini_parse_quantity(Z_STR_P(option), &errstr);
+			if (errstr) {
+				zend_error(E_WARNING, "%s", ZSTR_VAL(errstr));
+				zend_string_release(errstr);
+			}
+		} else if (Z_TYPE_P(option) == IS_LONG) {
+			result = Z_LVAL_P(option);
+		} else {
+			zend_value_error("Invalid %s value in $options argument", zend_get_type_by_const(Z_TYPE_P(option)));
```

Comment:
Since we are talking about values here, the `zend_zval_value_name()` function could provide a more precise result. BTW the majority of error messages were converted to use this function last year so I think it would be  beneficial to use it here as well.